Mod2/Lessons/Week5/CFUReview.md

```diff
@@ -0,0 +1,50 @@
+# Mod 2 Week 5 Checks for Understanding
+
+1. What are some reasons for using software versioning?
+
+1. How would you describe a "breaking change" that occurs in a software version?
+
+1. Can you think of a reason why you would want to use an older version as opposed to the most recent version?
```

Comment:
I think with these two questions, we might not need the first question.

Reply:
Totally agree, I think these two questions are more clear versions of the first question. Going to remove question 1.